api/src/main/java/org/openmrs/api/db/ObsDAO.java

```diff
@@ -109,5 +110,9 @@ public Long getObservationCount(List<Person> whom, List<Encounter> encounters, L
 			List<Concept> answers, List<PERSON_TYPE> personTypes, List<Location> locations, Integer obsGroupId,
 			Date fromDate, Date toDate, List<ConceptName> valueCodedNameAnswers, List<Visit> visits,
 			boolean includeVoidedObs, String accessionNumber) throws DAOException;
-	
+
+	/**
+	 * @see ObsService#getLatestObsByConceptId(String)
+	 */
+	Obs getLatestObsByConceptId(String conceptId);
```

Comment:
The advantage that i find with your above method over the existing one is that yours is more efficient. 😊

Reply:
Great, but too bad that I am not using it anymore